pkg/config/metadata_test.go

```diff
@@ -0,0 +1,63 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/RedHatGov/bundle/pkg/config/v1alpha1"
+)
+
+func Test_MetadataError(t *testing.T) {
+
+	path := "test"
+
+	tests := []struct {
+		name       string
+		createType string
+		want       error
+	}{
+		{
+			name:       "test diff error",
+			createType: "diff",
+			want:       fmt.Errorf("no metadata found at %s, please run a create full", path),
+		},
+		{
+			name:       "test full error",
+			createType: "full",
+			want:       fmt.Errorf("no metadata found at %s, please run a create diff", path),
+		},
+	}
+
+	for _, tt := range tests {
+
+		err := &MetadataError{Path: path, Type: tt.createType}
+
+		fmt.Print(err)
+
+		if err == tt.want {
```

Comment:
This will always evaluate to false, see https://play.golang.com/p/HF8UmmQxhmJ. Better to compare error strings here, or use [`require.Assert()`](https://pkg.go.dev/github.com/stretchr/testify/require?utm_source=godoc#Equal).

Reply:
TIL :) Changed to comparing the strings.